src/accounts/models.py

```diff
@@ -14,3 +14,18 @@ class CustomUser(AbstractUser):
 
     def __str__(self):
         return self.username
+
+
+class ServiceSeeker(models.Model):
+    user = models.OneToOneField(CustomUser, on_delete=models.CASCADE)
+    username = models.CharField(max_length=150, unique=True)
+    email = models.EmailField(unique=True)
+    bookmarked_services = models.JSONField(
+        default=list, blank=True, null=True
+    )  # Allow null values
+    location_preference = models.CharField(
+        max_length=255, help_text="Preferred borough or location in NYC"
+    )
+
```

Comment:
This will add to sqlite. Will we be proceeding with storing user data in sqlite3?